### DISCUSSION TITLE
How to set server_key to an empty string?

### DISCUSSION THREAD

**Discussion by pfaelzerchen** [lldap/lldap#1030] [Q&A] [answered]
There is a warning in my logs:

```
WARNING: A key_seed was given, we will ignore the server_key and generate one from the seed! Set server_key to an empty string in the config to silence this message.
```

I am running lldap on kubernetes, but without the pvc suggested in the kubernetes guide. I use the key_seed instead.

How do I set server_key to an empty string? I did not find something useful in the docs regarding this.

**Comment by nitnelave** [comment, marked as answer]
Huh, I guess the error could be clearer. Due to a historical artifact, the config key has 2 names: server_key and key_file. It's that one: https://github.com/lldap/lldap/blob/main/lldap_config.docker_template.toml#L109

You can set it either in the config or through env variables.

**Reply by nitnelave** [reply]
Can you create an issue to remind me to clarify the error?

**Reply by pfaelzerchen** [reply]
Thanks. I created the issue: https://github.com/lldap/lldap/issues/1032
